### ISSUE TITLE
MSYS2/Mingw-64: Unable to locate libedit

### ISSUE THREAD

**Issue by rootfather** [mgba-emu/mgba#1010]
Despite having the following packages installed, I get this error message on every cmake run:
```$ cmake .. -G "MSYS Makefiles" -DBUILD_STATIC=ON
-- Checking for one of the modules 'libedit'
CMake Warning at CMakeLists.txt:133 (message):
  Requested module libedit missing for feature USE_EDITLINE.  Feature
  disabled.
Call Stack (most recent call first):
  CMakeLists.txt:393 (find_feature)
```

I have the following libedit/editline related package installed:
```
msys/libedit 3.1-20170329
msys/libedit-devel
mingw64/mingw-w64-x86_64-wineditline
```

Is there anything that can be done about it? Also, is it intentional that the libepoxy and libelf packages are not mentioned in the MSYS2 section of the README file?

**Comment by endrift** [comment]
I was not aware that libedit works at all on Windows, so it's intentionally not used. I should probably disable it entirely on Windows

As for libelf, that's an oversight. Not sure about libepoxy offhand. If it compiles without it that may be fine.

**Comment by rootfather** [comment]
It compiles and works fine with and without libepoxy - not entirely sure where's the difference...

**Comment by endrift** [comment]
Do shaders work without libepoxy too?

**Comment by rootfather** [comment]
Good catch! No, with libepoxy either uninstalled or compiled with -DUSE_EPOXY=FALSE, the shader settings disappear completely from the Qt GUI.
